org.gradle.toolingapi/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,78 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Gradle Tooling API
+Bundle-Vendor: Gradle Inc.
+Bundle-SymbolicName: org.gradle.toolingapi
+Bundle-ClassPath: .
+Bundle-RequiredExecutionEnvironment: JavaSE-1.7
+Require-Bundle: org.slf4j.api;bundle-version="1.7.2"
+Export-Package: org.gradle;version="4.3.0",
```

Comment:
❌ Almost none of the packages below are part of the tooling API. We should only export the ones that are actually part of the official API.

Reply:
I didn't change anything in that regard. I remember we had problems before when not all packages were exported. IMO we should tackle this separately.

Reply:
Can you open an issue for that for 3.0? We definitely need to fix this, as these packages should not be exposed to anyone.

Reply:
https://github.com/eclipse/buildship/issues/671